static/themes.js

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2012-2017, Matt Godbolt
```

Comment:
Thanks for copying this...my employer prefers me to keep this on anything I do to prove they're not directly involved in it. Stupid legal. Feel free to add yourself to the name here though if you want! This is your file too!